apps/web-shop-front/src/pages/Home.tsx

```diff
@@ -1,60 +1,21 @@
-import { Card, Loader, Pagination } from '@team21/ui-components';
-import React, { FC, useCallback, useState } from 'react';
-import useGetProducts from '../graphql/useGetProducts';
-
-const PER_PAGE = 12;
+import { ProductTile, Loader } from '@team21/ui-components';
+import React, { FC, useEffect, useState } from 'react';
+import { Product } from '@team21/types'
+import useGetHomepageProducts from '../graphql/product/useGetHomepageProducts';
 
 const Home: FC<Record<string, never>> = () => {
-  const [currentPage, setCurrentPage] = useState(0);
-  const { data, loading } = useGetProducts(PER_PAGE, currentPage * PER_PAGE);
-
-  const onPageChange = useCallback(
-    (page: number) => {
-      setCurrentPage(page);
-    },
-    [setCurrentPage],
-  );
-
-  const total = data?.products_aggregate.aggregate.count;
+  const { data, loading } = useGetHomepageProducts(3);
 
-  if (total === 0) {
-    return <div className="flex items-center justify-center w-full h-full">No items...</div>;
-  }
+  if (!data && loading) return <Loader />;
 
-  //TODO style card a little bit better
-  return (
-    <div className="flex flex-col items-center justify-between w-full h-screen ">
-      {!data && loading && <Loader />}
-      {data && (
-        <div className="flex flex-wrap justify-between gap-3 max-w-7xl">
-          {data?.products.map((item) => (
-            <div key={item.id} className="min-w-max">
-              <Card
-                title={item.name}
-                body={
-                  <>
-                    <div>{item.description}</div>
-                    <div>Price: {item.price}</div>
-                    <div>Quantity: {item.quantity}</div>
-                  </>
-                }
-                buttonTitle={'view'}
-                imageSrc={item.image}
-              />
-            </div>
-          ))}
-        </div>
-      )}
-      <div className="w-full max-w-7xl">
-        <Pagination
-          currentPage={currentPage}
-          total={total ?? 0}
-          displayPerPage={PER_PAGE}
-          onPageChange={onPageChange}
-        />
-      </div>
+  return <div className="w-10/12 p-24 mx-auto">
+    <h1 className=" text-h1 text-center my-40 ml-24">Welcome to Web Shop!</h1>
+    <div className="flex justify-center items-center flex-wrap">
+        {data?.products && data?.products.map((product: Product, index: number)=>(
```

Comment:
check it, but i think you can use just: data?.products?.map?.()

Reply:
Will do.